package.json

```diff
@@ -49,6 +49,7 @@
     "karma-jasmine": "~3.0.1",
     "karma-jasmine-html-reporter": "^1.4.2",
     "protractor": "~5.4.3",
+    "ts-essentials": "^6.0.4",
```

Comment:
Adding a dependency here. I've never used ts-essentials but a coworker mentioned it and the name sounds promising :P

It has no dependencies